src/spyglass/position/v1/position_dlc_selection.py

```diff
@@ -65,69 +65,84 @@ def make(self, key):
         pos_nwb = (DLCCentroid & key).fetch_nwb()[0]
         ori_nwb = (DLCOrientation & key).fetch_nwb()[0]
 
-        pos_obj = pos_nwb["dlc_position"].spatial_series["position"]
-        vel_obj = pos_nwb["dlc_velocity"].time_series["velocity"]
-        vid_frame_obj = pos_nwb["dlc_velocity"].time_series["video_frame_ind"]
-        ori_obj = ori_nwb["dlc_orientation"].spatial_series["orientation"]
-
-        position = pynwb.behavior.Position()
-        orientation = pynwb.behavior.CompassDirection()
-        velocity = pynwb.behavior.BehavioralTimeSeries()
-
-        position.create_spatial_series(
-            name=pos_obj.name,
-            timestamps=np.asarray(pos_obj.timestamps),
-            conversion=pos_obj.conversion,
-            data=np.asarray(pos_obj.data),
-            reference_frame=pos_obj.reference_frame,
-            comments=pos_obj.comments,
-            description=pos_obj.description,
-        )
+        if isinstance(pos_nwb["dlc_position"], pd.DataFrame):
```

Comment:
I think I would migrate these to a separate method to clean up the indenting, something like `declare_pos_analysis_file`